libchisel/src/deployer.rs

```diff
@@ -65,27 +66,26 @@ fn deployer_code() -> Vec<u8> {
 }
 
 /// Returns a module which contains the deployable bytecode as a custom section.
-fn create_custom_deployer(payload: &[u8]) -> Module {
+fn create_custom_deployer(payload: &[u8]) -> Result<Module, ModuleError> {
```

Comment:
If we start to see `Result<Module, ModuleError>` everywhere we should include a `type Result<T> = ::std::result::Result<T, ModuleError>` in `lib.rs`